src/cli/command_list.py

```diff
@@ -118,18 +120,32 @@ def applications_list(
     ),
 ) -> None:
     """
-    List applications.
+    List applications available to the user.
+
+    Args:
+        remote: Boolean flag to list remote applications
+        local: Boolean flag to list local applications
     """
     if not remote and not local:
         remote = local = True
 
-    if remote:
-        typer.echo("List applications from Quantum Network Explorer.")
-    if local:
-        typer.echo("List applications from disk.")
     applications = processor.applications_list(remote=remote, local=local)
-    for application in applications:
-        typer.echo(application)
+
+    if 'local' in applications:
```

Comment:
Why not a nice overview (columns aligned of course):
NAME                  ID                              LOCATION                       PATH
bb-84                   1                               local                               ~\.qne\workspace\bb-84
bb-85                   2                               local                               ~\.qne\workspace\bb-84
bb-86                   11                             remote

Reply:
Added a tabular output for the list command